st3/lsp_utils/node_runtime.py

```diff
@@ -11,17 +11,18 @@
 import os
 import shutil
 import sublime
+import subprocess
 import sys
 import tarfile
 import urllib.request
 import zipfile
 
 __all__ = ['NodeRuntime', 'NodeRuntimePATH', 'NodeRuntimeLocal']
 
-IS_MAC_ARM = sublime.platform() == 'osx' and sublime.arch() == 'arm64'
 IS_WINDOWS_7_OR_LOWER = sys.platform == 'win32' and sys.getwindowsversion()[:2] <= (6, 1)  # type: ignore
 
-DEFAULT_NODE_VERSION = '16.15.0'
+DEFAULT_NODE_VERSION = '16.17.1'
```

Comment:
The [latest v16 release](https://nodejs.org/dist/latest-v16.x/) is v16.19.1.

Reply:
I've set this version because it matches the version that electron ships with.
When using electron the plain node.js is still used for installing dependencies so it's best to use the same one.

Reply:
make sense.